std/math.d

```diff
@@ -4518,7 +4518,7 @@ real round(real x) @trusted nothrow @nogc
     {
         auto old = FloatingPointControl.getControlState();
         FloatingPointControl.setControlState(
-            (old & ~FloatingPointControl.ROUNDING_MASK) | FloatingPointControl.roundToZero
+            (old & (-1-FloatingPointControl.ROUNDING_MASK)) | FloatingPointControl.roundToZero
```

Comment:
Why not a cast instead?

Reply:
I like this better, as it will invoke the compiler's normal integral promotion. I try to avoid casts if I can.

Reply:
spaces around binary `-`...